src/utils/defaultLanguageStrings.ts

```diff
@@ -76,13 +76,15 @@ export default {
   COLUMN_DEADLINE_DATE_TOOLTIP: 'Deadline for applicant to respond',
   COLUMN_APPLICATION: 'Application',
   COLUMN_APPLICATION_REVIEW: 'Application',
-  COLUMN_APPLICANT: 'Applicant',
+  COLUMN_APPLICANT: '',
   COLUMN_ORGANISATION: 'Organisation',
+  COLUMN_REVIEWER: '',
   COLUMN_CONSOLIDATOR: 'Consolidator',
   COLUMN_STAGE: 'Stage',
   COLUMN_STATUS: 'Status',
   COLUMN_OUTCOME: 'Outcome',
   COLUMN_REVIEWER_ACTION: '',
+  COLUMN_ASSIGNER_ACTION: '',
```

Comment:
Just wondering why the empty strings?

If we're not using them, why not remove them completely, or if you want to keep them around for completion sake, then we may as well have proper values in them, no?

Reply:
Oh, yea. Just forgot to remove these. There are some like that though, but it's because we want to have the column, and a title is expected, but it doesn't display any text currently...

Reply:
Okay cool, that makes sense.